apps/utils/scripts/build-bfd

```diff
@@ -145,8 +148,8 @@ args=""
 [[ $use_minio == "true" ]] && args="$args -Ds3.local=true"
 [[ $projects != "" ]] && args="$args$projects"
 [[ $include_dependencies = "true" ]] && args="$args -am"
-[[ $test_arg != "" ]] && args="$args -Dtest=\"$test_arg\""
-[[ $it_test_arg != "" ]] && args="$args -Dit.test=\"$it_test_arg\""
+[[ $test_arg != "" ]] && args="$args -Dtest=$test_arg"
```

Comment:
All the changes besides setting the timezone are things shellcheck was complaining about